subprojects/docs/src/docs/userguide/buildingJavaProjects.adoc

```diff
@@ -346,11 +346,11 @@ If you instead want to create an 'uber' (AKA 'fat') JAR, then you can use a task
 
 There are several options for publishing a JAR once it has been created:
 
- * the `uploadArchives` task — the <<artifact_management,original publishing mechansim>> — which works with both Ivy and (if you apply the <<maven_plugin,Maven Plugin>>) Maven
  * the <<publishing_maven,Maven Publish Plugin>>
  * the <<publishing_ivy,Ivy Publish Plugin>>
+ * the `uploadArchives` task — the <<artifact_management,original publishing mechanism>> — which works with both Ivy and (if you apply the <<maven_plugin,Maven Plugin>>) Maven
```

Comment:
Is this mechanism now officially deprecated? If so, we should call that out here and explain that it's only included for historic reasons or something similar. I want it to be very clear that people should not be using it if that's the case. I don't feel that just saying that the other two options are preferred is sufficient.

Reply:
> Is this mechanism now officially deprecated?

Not yet, it will be once #4940 is resolved. I'll add a note there.